porch/func/wrapper-server/main.go

```diff
@@ -95,14 +96,16 @@ func (e *singleFunctionEvaluator) EvaluateFunction(ctx context.Context, req *pb.
 	cmd.Stdout = &stdout
 	cmd.Stderr = &stderr
 
+	start := time.Now()
+
 	err := cmd.Run()
 	var exitErr *exec.ExitError
 	if err != nil && !errors.As(err, &exitErr) {
 		return nil, status.Errorf(codes.Internal, "Failed to execute function %q: %s (%s)", req.Image, err, stderr.String())
 	}
 
 	outbytes := stdout.Bytes()
-	klog.Infof("Evaluated %q: stdout length: %d\nstderr:\n%v", req.Image, len(outbytes), stderr.String())
+	klog.Infof("Evaluated %q: duration=%v\nstdout length: %d\nstderr:\n%v", req.Image, time.Since(start).Round(time.Millisecond), len(outbytes), stderr.String())
```

Comment:
This latency is not the user-facing latency. When we calculate the time here, pod is already running.
If the KRM function is not in the cache, we will need to create a pod.